content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,6 +38,30 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
+## Documentation content standards
+
+Kubernetes documentation does not host certain types of content:
+
+* Vendor, provider, or project-specific content from projects that are not:
+
+    * Inside the kubernetes or kubernetes-sigs GitHub organizations
+    * Inside the CNCF
+
+* Identical content hosted in multiple locations ("dual-sourced" content)
+
+For example:
+
+| Allowed |                           Action                          |                                                    Reason                                                   |
+|:-------:|:----------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+|    Y    | Linking to Kubernetes in Docker (KinD) docs from k8s docs | The KinD project resides in github.com/kubernetes-sigs.                                                     |
+|    N    | Hosting KinD docs in k8s website                          | Dual-sourced content requires duplicated effort from maintainers and tends to go stale/rotten more quickly. |
+|    Y    | Linking to Prometheus docs from k8s docs                  | Prometheus is a CNCF project.                                                                                |
+|    N    | Linking to rkt docs from k8s docs                         | rkt is an (almost) archived CNCF project.                                                                    |
+|    N    | Tutorial on using a specific product to monitor Kubernetes | Third-party product-specific content not allowed; this type of content belongs in the third-party product's documentation. |
+|    Y    | Linking to vendor-specific product docs when listing third-party products such as Ingress controllers, load-balancing products, monitoring tools, and device plugins | Avoids dual-sourced content
+|    Y    | Linking to an online training course                      |  Avoids dual-sourced content                                                                                  |
```

Comment:
There's maybe [one page](https://github.com/kubernetes/website/blob/master/content/en/docs/setup/_index.md) where we accept this kind of content. Consequently, please omit this example.

Reply:
[Tutorials/OnlineTrainingCources/Overview...](https://kubernetes.io/docs/tutorials/online-training/overview/) lists close to 30 online training courses. So I think it would be better to change "Linking to an online training course" to N with an appropriate  reason.